docs/data-builder-quick-start.md

```diff
@@ -0,0 +1,55 @@
+There are 3 components that need to be added to a project to use 
+Databuilder. These are:
+
+1. Add Data Builder to a `project.yaml`
+2. Add a Dataset definition
+3. Specify a dummy data file. 
```

Comment:
question: I wrote this section early on as a placeholder.

There are now several ways of running Data Builder, even with the OpenSAFELY CLI:

1. `opensafely run <action>`
2. `opensafely exec databuilder …`

Which one should we be advocating? Probably the one that's already documented, maybe?

This page probably needs linking to somewhere that describes running an OpenSAFELY project.